godot-macros/src/class/godot_api.rs

```diff
@@ -47,7 +47,10 @@ pub fn attribute_godot_api(input_decl: Declaration) -> Result<TokenStream, Error
 
 /// Attribute for user-declared function
 enum BoundAttrType {
-    Func { rename: Option<String> },
+    Func {
+        rename: Option<String>,
+        gd_self: bool,
```

Comment:
Maybe `has_gd_self`; this sounds like it contains the value of `gd_self`.

Reply:
Yeah, definitely. I had expected this name to be more temporary, but then `gd_self` got kept as the attribute name so I never cleaned this up.